static/app/components/events/eventTags/index.tsx

```diff
@@ -101,13 +101,14 @@ export function EventTags({event, organization, projectSlug, location}: Props) {
   const streamPath = `/organizations/${orgSlug}/issues/`;
   return (
     <StyledClippedBox clipHeight={150}>
-      {shouldDisplayTagTree ? (
+      {shouldUseNewTagsUI() ? (
```

Comment:
since this is a hook it should be higher up and follow the naming for hooks
```suggestion
      {useShouldDisplayNewTagsUI() ? (
```